plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/LocalAppEngineConsolePageParticipant.java

```diff
@@ -37,6 +38,12 @@
 public class LocalAppEngineConsolePageParticipant implements IConsolePageParticipant {
   private LocalAppEngineConsole console;
   private Action terminateAction;
+  private IServerListener serverStateListener = new IServerListener() {
+    @Override
+    public void serverChanged(ServerEvent event) {
+      update();
```

Comment:
I noticed that `update()` calls `Behavior.canStop()`. Looks like we're not really supposed to call `canStop()` according to its javadoc:

```
	 * Clients should never call this method directly.
```

If we never call this, then we don't need to provide the trivial implementation of `Behavior.canStop()` in above, because

```
	 * This method is called by the server core framework,
	 * in response to a call to <code>IServer.canStop()</code>.
	 * The framework has already filtered out obviously invalid situations,
	 * such as stopping a server that is already stopped.
```

Random guess, but if we call `IServer.canStop()` instead, we may be able to remove `Behavior.canStop()`.

But I guess this may be out of the scope of this PR.